scripts/code_coverage_report/README.md

```diff
@@ -0,0 +1,37 @@
+# Code Coverage Report Generation
+
+This tool is to help generate coverage reports for pull requests in the [test_coverage  workflow](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/code_coverage_report/code_coverage_file_list.json).
```

Comment:
Alternative to the first paragraph:

This tool is to help generate coverage reports for pull requests. It's defined by the [test_coverage  workflow](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/test_coverage.yml)